### DISCUSSION TITLE
Adding another command Docker Compose

### DISCUSSION THREAD

**Discussion by avenbreaks** [0xPolygon/polygon-edge#835] [Ideas]
hello can polygon team adding another command for built genesis json like 
--pos 
--consensus 
etc 

_https://wiki.polygon.technology/docs/edge/get-started/cli-commands#genesis-flags_

`command: ["-mode","init", "-num-nodes", "10", "-premine", "address:total-amount"]` 

i'm trying to create using docker file but some command not found for deploying

**Comment by laviniat1996** [comment]
Hi @avenbreaks,
What command are you asking to be added?
